version/derive/tests/ui_fail/unknown_field_in_version_attribute.stderr

```diff
@@ -1,10 +1,8 @@
-error: Unknown field
- --> tests/ui_fail/unknown_field_in_version_attribute.rs:7:1
+error: Unknown field: `derive`
+ --> tests/ui_fail/unknown_field_in_version_attribute.rs:7:50
   |
 7 | #[version(n = 1, versioned = "VersionedMessage", derive = "Clone")]
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-  |
-  = note: this error originates in the attribute macro `version` (in Nightly builds, run with -Z macro-backtrace for more info)
+  |                                                  ^^^^^^^^^^^^^^^^
```

Comment:
How far can we push `manyhow`. It looks like we would be able to provide more context and more details in the error message. 

E.g. `unknown field: ``derive`` `, the macro `version` can only accept `n`, `versioned` (see previous comment about naming convention.

Reply:
Hmm.. I think it's possible to provide a manual hint with what the `#[version]` macro should look like.

Reply:
tbh, I don't think it's a good idea. If we just add a blanket reminder what the attribute should look like for any error, it risks giving irrelevant messages.

Reply:
This is not a hard requirement, I'll approve regardless of if it's implemented. 

However, AFAICT, a lot of our attributes are simple key-value pairs and for keys which are known ahead of time. For this use case, all you need to do is add a generic method to show a message for the recognised attributes. If all this does is print `help` and each such proc macro has a `help` message it does the job.

Reply:
Created #3736